routes/index.js

```diff
@@ -62,9 +62,17 @@ module.exports = function(utils, env, nunjucksEnv, appName) {
         }));
       }
 
+      //We add the localization code to the query params through a URL object
+      //and set search prop to nothing forcing query to be used during url.format()
+      var thimbleUrl;
+      var urlObj = url.parse(req.url, true);
+      urlObj.search = "";
```

Comment:
Why do you clear this?  Won't we lose things like `?enableExtensions=...` and the like?